src/Symfony/Bundle/SecurityBundle/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 CHANGELOG
 =========
 
+3.3.0
+-----
+
+ * Deprecated registering by convention the `UserPasswordEncoderCommand` command 
```

Comment:
It's more about extending the command and registering it from another bundle actually. Can be rephrased I guess.